File: app/src/main/java/com/example/downloadsimple/Java/DownloadListener.java
package com.example.downloadsimple.Java;

/**
 * Created by 张垚杰 on 2018/1/16.
 */

public interface DownloadListener {
    void onProgress(int progress);
    void onSuccess();
    void onFailed();
    void onPaused();
    void onCanceled();
}

File: app/src/main/java/com/example/downloadsimple/Kotlin/DownloadListener.kt
package com.example.downloadsimple.Kotlin

/**
 * Created by 张垚杰 on 2018/1/16.
 */
interface DownloadListener {
     fun onProgress(progress: Int?)
     fun onSuccess()
     fun onFailed()
     fun onPaused()
     fun onCanceled()
}
